tests/migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -204,9 +204,9 @@ sub run {
     }
     assert_screen 'yast2-migration-target';
     send_key "alt-p";                                       # focus on the item of possible migration targets
-    send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 5;
-    wait_still_screen 5;
-    send_key "alt-n";
+    wait_screen_change(undef, 20);
```

Comment:
please don't do that. There is a reason why `wait_screen_change` uses positional arguments and the first parameter should not be ignored. You could easily wrap the former line within, e.g.

```perl
- send_key 'alt-p';
+ wait_screen_change { send_key 'alt-p' }, 20;
```

but then the `wait_screen_change` is just a lazy `assert_screen` when you haven't specified the expected screen. If you are convinced that we need to wait after the key press of 'alt-p' so see a screen ready to go further down afterwards then you should add an explicit `assert_screen` in the middle. The whole code section here can then look like the following:

```
assert_screen 'yast2-migration-target';
send_key 'alt-p';
assert_screen 'yast2-migration-target-list-selected';
send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
send_key 'alt-n';
```

Reply:
Verified the first method. It can work now.